xalia/main.gudl

```diff
@@ -495,7 +495,7 @@ root (descendent_matches(in_list_navigation)) {
 }
 
 // Special win32 controls
-if (class_name == "SysListView32") {
+if (class_name == "SysListView32" and not is_hwnd_element) {
     win32_use_listview: ancestor_matches(toplevel_interactable);
     targetable: false;
     recurse: false;

```

Comment:
It has to do with the 2 backends co-existing. The old backend uses this to ignore the UIA elements for these and create Win32 element children. Since the new backend supports class_name, this condition applies to it and would cause us to ignore these controls.

There's a lot that can be simplified in gudl when we lose the old backend, I think.

Reply:
Got it, that makes sense. This PR looks good to me then.